src/components/posts/Comments.js

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from 'react'
+import CommentItem from './CommentItem'
+import FilterMenu from './utils/filterMenu'
+
+const Comments = ({ post, setUser }) => {
+  const [sortType, setSortType] = useState('Most Recent')
+  const [comments, setComments] = useState(post.comments)
+  const [showingAll, setShowingAll] = useState(false);
+
+  useEffect(() => {
+    setComments(post.comments)
+    handleSorting()
+    // eslint-disable-next-line
+  }, [sortType, post])
+
+  const handleSorting = () => {
+    if (sortType === 'Most Recent') {
+      const sortedComments = [...post.comments]
+      setComments(sortedComments.sort((a, b) => {
```

Comment:
I don't think it's necessary to put them into a new array, you can just pass `post.comments.sort(` directly into the `setComments` function